docs/source/distributions.rst

```diff
@@ -262,9 +269,9 @@ LeakyReLUTransform
     :undoc-members:
     :show-inheritance:
 
-PermuteTransform
-----------------
-.. autoclass:: pyro.distributions.transforms.PermuteTransform
+Permute
```

Comment:
some of these changes probably need deprecation warnings? @fritzo can you please point stefan to a good example pattern to follow?

Reply:
IIUC think these have been renamed without making redirects, and I am ok with that. If you believe any of these flows have important existing users, you could add a redirect
```py
def PermuteTransform(*args, **kwargs):
    warnings.warn("PermuteTransform has been renamed to Permute",
                  DeprecationWarning)
    return Permute(*args, **kwargs)
```
However I think its fine to omit these redirects.

Reply:
Okay, I'll omit these!